src/helpers.php

```diff
@@ -18,6 +18,22 @@ function cp_route($route, $params = [])
     return $route;
 }
 
+function api_route($route, $params = [])
```

Comment:
Can we call this `statamic_api_route` or `Statamic\Statamic::apiRoute()`?

Feels generic enough that it could conflict with someone's app pretty easily.

Reply:
Or trait?  Or do you think the helper is something other people will use?

Reply:
I think putting it in `Statamic` is enough. We have `isApiRoute` in there, it's pretty similar.